tests/functional/base_pages/question.page.js

```diff
@@ -35,6 +35,7 @@ class QuestionPage extends BasePage {
 
   switchLanguage(language_code) { return 'a[href="?language_code=' + language_code + '"]'; }
 
+  hubLink() { return 'a[href="/questionnaire/"]'; }
```

Comment:
Sorry to be petty, can we call this `returnToHubLink` as it makes me think its a link on the hub.

Reply:
Changed it, thanks!